src/lib/__tests__/utils-test.js

```diff
@@ -342,6 +342,45 @@ describe('utils.getRefinements', () => {
     );
   });
 
+  it('should retrieve one facetRefinement and inject query facet if clearQuery === true', () => {
+    helper.toggleRefinement('facet1', 'facet1val1');
+    helper.setQuery('my query');
+    const expected = [
+      {
+        type: 'query',
+        attributeName: 'query',
+        name: 'my query',
+        query: 'my query',
+      },
+      { type: 'facet', attributeName: 'facet1', name: 'facet1val1' },
+    ];
+    const clearsQuery = true;
+    expect(
+      utils.getRefinements(results, helper.state, clearsQuery)
+    ).toHaveLength(2);
+    expect(
+      utils.getRefinements(results, helper.state, clearsQuery)
+    ).toContainEqual(expected[0]);
```

Comment:
Why using `toContainEqual` on each elements rather than `isEqual` on the global array?

Reply:
isEqual adds a constraint of order which I didn't think was relevant.

Reply:
since I simplified the tests and removed the other refinement, I now use isEqual